### PR TITLE
Add pidof

### PR DESCRIPTION
**Pull request by asdfugil**
### All Submissions

* [x] Have you made sure there aren't any other open [Pull Requests](https://github.com/ProcursusTeam/Procursus/pulls) for the same update/change?
* [ ] This Pull Request doesn't contain any package additions; it's a small change (e.g README change)

### Package Additions/Updates

* [x] Have you confirmed this builds & works as intended on an iOS device (if applicable)?
* [ ] Have you confirmed this builds & works as intended on a macOS device (if applicable)?

checksums from http://www.nightproductions.net/hashes/pidof_source.txt

### CODE REVIEW

**Comment by CRKatri** [comment]
This does not belong in the bootstrap.

**Comment by m1stadev** [comment]
> This does not belong in the bootstrap.

ratio + L bozo

**Review by m1stadev — APPROVED** [review]

File: makefiles/pidof.mk [hunk 1]
```diff
@@ -0,0 +1,46 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+STRAPPROJECTS += pidof
```
Comment (CRKatri) [inline makefiles/pidof.mk:5]: This is a `SUBPROJECT` not a `STRAPPROJECT`

**Review by CRKatri — CHANGES_REQUESTED** [review]

**Comment by asdfugil** [comment]
PROTIP: This has similar importance to `vi` if not more which also does not belong to the bootstrap so no one should ever try to add `vi` to the bootstrap.

**Comment by CRKatri** [comment]
`ps aux | grep "process name"` seems kinda useless to me. It's also not on macOS or FreeBSD base installs, so I see no need for it to be part of the strap. It's importance is way less than `vi`, `vi` serves a legitimate purpose and deserves to be in the strap, but is not to keep the size down.
This is not going in the strap. Period.

File: makefiles/pidof.mk (hunk 1 above)
Comment (asdfugil) [reply makefiles/pidof.mk]: All right

**Comment by asdfugil** [comment]
It's useful for `renice(1)` but ok.

**Pull request closed without merging**